.github/workflows/main.yml

```diff
@@ -0,0 +1,41 @@
+name: Main
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  all:
+    runs-on: windows-latest
+    timeout-minutes: 10
+
+    steps:
+
+      - uses: Brightspace/third-party-actions@actions/checkout
+      - uses: Brightspace/third-party-actions@actions/setup-dotnet
+        with:
+          dotnet-version: 7.0.x
+      - uses: Brightspace/third-party-actions@NuGet/setup-nuget 
+ 
+      - name: Add msbuild to PATH
+        uses: microsoft/setup-msbuild@v1.1

```

Comment:
I'm hoping to find an alternative to `msbuild`, but if not I will add it to our third-party-actions repo before merging this.